app/chewy/statuses_index.rb

```diff
@@ -3,7 +3,7 @@
 class StatusesIndex < Chewy::Index
   include FormattingHelper
 
-  settings index: { refresh_interval: '30s' }, analysis: {
+  settings index: index_preset(refresh_interval: '30s', number_of_shards: 5), analysis: {
```

Comment:
I'm running a single ES instance. Is the default 5 shard correct in that situation?

Reply:
Yes, it will improve performance even on single-node ES instance as each search uses a thread per shard. Also shard sizes began to be over the maximum size recommended by ES authors, even on single-node servers